src/lib/in.c

```diff
@@ -1768,6 +1742,12 @@ build_cflow_automaton(inputctx* ictx){
     { "[E", simple_cb_begin, },
     { "[<\\N;\\N;\\NM", mouse_press_cb, },
     { "[<\\N;\\N;\\Nm", mouse_release_cb, },
+    { "[<\\N;-\\N;\\NM", mouse_press_cb, },
+    { "[<\\N;-\\N;\\Nm", mouse_release_cb, },
+    { "[<\\N;\\N;-\\NM", mouse_press_cb, },
+    { "[<\\N;\\N;-\\Nm", mouse_release_cb, },
+    { "[<\\N;-\\N;-\\NM", mouse_press_cb, },
+    { "[<\\N;-\\N;-\\Nm", mouse_release_cb, },
```

Comment:
erp, i need some unit tests on this badly. i know touching it sometimes has unexpected results, due to a bug or bugs of mine.

Reply:
I've been using it daily for a few months without noticing any wierdness, but yeah, it didn't look like the "correct" fix to me.